### ISSUE TITLE
plugin "render as JSON" returns text/html in 3.3.0.RC1

### ISSUE THREAD

**Issue by sheehan** [grails/grails-core#10745]
The returned `Content-Type` is incorrect in 3.3.0.RC1. Works correctly in previous versions. 

The controller action: https://github.com/sheehan/grails-console/blob/master/grails3/plugin/grails-app/controllers/org/grails/plugins/console/ConsoleController.groovy#L52. 

I haven't been able to narrow it down yet.

### Steps to Reproduce

1. create new app
2. add dependency `runtime 'org.grails.plugins:grails-console:2.0.9'`
3. go to `/console`
4. execute code

### Expected Behaviour

`/console/execute` returns "application/json" Content-Type.

### Actual Behaviour

`/console/execute` returns "text/html" Content-Type.

### Environment Information

- **Operating System**: Mac
- **Grails Version:** 3.3.0.RC1
- **JDK Version:** 1.8
- **Container Version (If Applicable):** N/A

### Example Application

Described above.

**Comment by graemerocher** [comment]
A simple test: 

```groovy
class TestController {

    def index() { 
		def test = [foo:'bar']

		render test as JSON
    }
}
```

Results in 

```
HTTP/1.1 200 
X-Application-Context: application:development
Content-Type: application/json;charset=UTF-8
Transfer-Encoding: chunked
Date: Mon, 24 Jul 2017 15:05:40 GMT

{"foo":"bar"}
```

**Comment by sheehan** [comment]
It seems like it only happens if the plugin was created with < 3.3. I created a plugin using 3.2.11 with the above controller and included it's jar in my 3.3 app. 

I get this:
```
HTTP/1.1 200
X-Application-Context: application:development
Content-Type: text/html;charset=UTF-8
Content-Length: 13
Date: Sat, 29 Jul 2017 13:24:26 GMT

{"foo":"bar"}
```

**Comment by codeconsole** [comment]
Are 3.2.x plugins meant to be compatible with Grails 3.3?
I've had to recompile my private 3.2.x plugins to get them to work due to various issues.

**Comment by jphiloon** [comment]
Fixed for me by updating console plugin to 2.1.1.  
However, I notice that JSON output does not wrap.  
For instance, something like

    def users = Users.list()
    users[0] as JSON

results in truncated output.  Long strings themselves do wrap, so not sure what the difference is

**Comment by ilopmar** [comment]
Fixed using latest console plugin.
